src/main/resources/db/migration/V14__schedlock.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE shedlock (
```

Comment:
Kan vi kalle den for midlertidig_utilgjengelig_shedlock, eller noe annet, i tilfelle vi vil ha en annen shedlock på et tidspunkt?

Reply:
Blir det ikke samme tabell, men med annet navn hvis det skjer?

Reply:
Jeg finner ikke noe dokumentasjon som sier akkurat det Joar. Men jeg synes det høres logisk ut.
LockProvideren som bruker denne tabellen gjelder vel for alle låsene vi ønsker.
De går vel ikke i beina på hverandre så lenge vi bruker forskjellige `name` her: `@SchedulerLock(name = "oppdaterMidlertidigUtilgjengelig")`